test/turn_test.rb

```
@@ -0,0 +1,56 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/card'
+require './lib/turn'
+
+class TurnTest < Minitest::Test
+
+  def test_it_exists
+    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn = Turn.new("guess", card)
+
+    assert_instance_of Turn, turn
+  end
+
+  def test_it_has_a_card
+    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn = Turn.new("guess", card)
+
+    assert_equal card, turn.card
+  end
+
+  def test_it_has_a_guess
+    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn = Turn.new("guess", card)
+
+    assert_equal "guess", turn.guess
+  end
+
+  def test_guess_can_be_correct
+    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn = Turn.new("Juneau", card)
+
+    assert_equal true, turn.correct?
+  end
+
+  def test_guess_can_be_incorrect
+    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn = Turn.new("Anchoridge", card)
+
+    assert_equal false, turn.correct?
+  end
+
+  def test_it_has_feedback
+    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn = Turn.new("Juneau", card)
+
+    assert_equal "Correct!", turn.feedback
+  end
+
+  def test_it_has_incorrect_feedback
+    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn = Turn.new("Anchoridge", card)
+
+    assert_equal "Incorrect.", turn.feedback
```

These last two tests are failing because you are using puts in your Turn, rather than returning theses strings